textstat/textstat.py

```diff
@@ -181,23 +181,22 @@ def linsear_write_formula(self, text):
         difficult_word = []
         text_list = text.split()
 
-        Number = 0
         for i, value in enumerate(text_list):
             if i <= 101:
                 try:
                     if self.syllable_count(value) < 3:
                         easy_word.append(value)
                     elif self.syllable_count(value) > 3:
                         difficult_word.append(value)
-                    text = ' '.join(text_list[:100])
-                    Number = float((len(easy_word)*1 + len(difficult_word)*3)/self.sentence_count(text))
-                    if Number > 20:
-                        Number /= 2
-                    else:
-                        Number = (Number-2)/2
                 except Exception as E:
                     print("Error (LWF): ", E)
-        return float(Number)
+        text = ' '.join(text_list[:100])
```

Comment:
The `try..except..` could do with moving to this section as well, as this is where the calculation is happening.

Reply:
I assumed that the exception can't occur on the calculation part. Since there is no critical calculation like divide by zero possible. So I guess it should be sufficient on the other part? 
Thanks